### PR TITLE
Add content item coverage to dump-data

### PR DESCRIPTION
**Pull request by emmabeynon**

### CODE REVIEW

File: bin/dump-data [hunk 1]
```diff
@@ -52,6 +52,15 @@ def find_working_example(schema_name, document_type, rendering_app, publishing_a
   nil
 end
 
+def count_content_items(schema_name, document_type)
+  puts "Counting content items with schema_name: #{schema_name} and document_type: #{document_type}"
+
+  ContentItem.where(
```
Comment (thomasleese) [inline bin/dump-data]: I think we need to include `rendering_app` and `publishing_app` in the where clause here as I define them to be part of the format in this matrix.

File: generate-matrix [hunk 2]
```diff
@@ -14,3 +14,7 @@ csv_file=matrix_$(date +%F_%T).csv
 ods_file=matrix_$(date +%F_%T).ods
 
 scp content-store-1.integration:/var/apps/content-store/dump.csv ./$csv_file
+
```
Comment (thomasleese) [inline generate-matrix:17]: Would it be possible to split this up into a separate commit?

File: generate-matrix [hunk 3]
```diff
@@ -14,3 +14,7 @@ csv_file=matrix_$(date +%F_%T).csv
...
+echo Removing dump and dump script from integration
+
+rm ./dump-data && rm ./$csv_file
```
Comment (thomasleese) [inline generate-matrix]: Also this will remove the files from your local machine, not integration. You'll want to do something like `ssh content-store-1.integration 'rm ...'` like above.

File: bin/dump-data [hunk 4]
```diff
@@ -97,6 +109,7 @@ def generate_dump
       frontend_status,
       backend_status,
       example_url,
+      content_item_coverage
```
Comment (thomasleese) [inline bin/dump-data]: Sorry one more change, can you move this above the example url field and I'll update the template spreadsheet.

File: generate-matrix [hunk 5]
```diff
@@ -14,3 +14,7 @@ csv_file=matrix_$(date +%F_%T).csv
 ods_file=matrix_$(date +%F_%T).ods
 
 scp content-store-1.integration:/var/apps/content-store/dump.csv ./$csv_file
+
+echo Removing dump and dump script from integration
+
+ssh content-store-1.integration 'rm ./dump-data && rm ./$csv_file'
```
Comment (thomasleese) [inline generate-matrix]: Ooh, I've just noticed I'm not sure if this will work since it needs to remove it from the `/var/apps/content-store` folder which I don't think it goes to by default when you SSH in.

Comment (emmabeynon) [reply generate-matrix]: Yep, just realised this after running the script again! 👍

**Review by thomasleese — APPROVED** [review]
